### ISSUE TITLE
Install instructions

### ISSUE THREAD

**Issue by bingalls** [geier/choose#4]
Consider adding install instructions to readme.md. In particular,
`sudo pip install urwid`
Ez-install system could also work. System should prompt for syntax to install pip, as needed.

**Comment by geier** [comment]
Good idea, I probably also should add a setup.py or even upload it to PyPI to make it even easier to install.
